stdlib/private/StdlibCollectionUnittest/CheckSequenceType.swift

```diff
@@ -127,7 +127,11 @@ public struct FindTest {
   ) {
     self.expected = expected
     self.element = MinimalEquatableValue(element)
-    self.sequence = sequence.map(MinimalEquatableValue.init)
+    var elementIndex = 0
+    self.sequence = sequence.map {
+    	defer { elementIndex += 1 }
```

Comment:
It is strongly discouraged to write side-effectful closures for map().  I'd recommend `sequence.enumerated().map { ... }`.